flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.flink.streaming.api.functions.source.SourceFunction
 import org.apache.flink.streaming.api.scala.DataStream
 import org.apache.flink.streaming.api.watermark.Watermark
 import org.apache.flink.table.api.{DataTypes, Schema}
-import org.apache.flink.table.api.Expressions.$
+import org.apache.flink.table.api.Expressions.{$, withAllColumns}
```

Comment:
Is this import used?

Reply:
Good catch
I 've again forgot that spotless does not clean unused imports in scala...
I will add commit to clean it once `test_ci connect` is finished on ci to be sure that the issue fixed

Reply:
:( now we're in a situation, we need to squash the two fixes 🤦

Reply:
could it be cherry-picked to one of the PRs and merged nothe commits 
or you mean squash both fixes in one commit?

Reply:
> could it be cherry-picked to one of the PRs and merge the commits
or you mean squash both fixes in one commit?

separate commits, absolutely. I meant to squash PRs 😅